app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1615,6 +1615,14 @@ impl GraphEditorModelWithNetwork {
             let profiling_max_duration              = &self.model.profiling_statuses.max_duration;
             node.set_profiling_max_global_duration <+ self.model.profiling_statuses.max_duration;
             node.set_profiling_max_global_duration(profiling_max_duration.value());
+
+
+            // === Panning camera to created node ===
+
+            pos_update_count <- node.output.position.count();
+            first_update_of_pos <- pos_update_count.filter(|i| *i==1).constant(());
+            first_update_of_pos_if_editing <- first_update_of_pos.gate(&self.frp.node_editing);
+            eval first_update_of_pos_if_editing([model](_) model.pan_camera_to_node(node_id));
```

Comment:
Here should be an exhaustive explanation why we're doing this magic.

And it would be nice to drop those nodes once the camera will be panned, so we don't increase the counter needlessly over many nodes. This is not trivial, however: the best way of doing it is
* creating a new frp::Network and put it into Rc<RefCell<>>
* creating those nodes in a bridge network between self.network, node_network and the special network.
* In `eval first_update_of_pos_if_editing` we drop the special network.